rpc.go

```diff
@@ -126,21 +126,24 @@ func (r rpc) Channel(service string) transport.Channel {
 }
 
 func (r rpc) Start() error {
-	callServe := func(i transport.Inbound) func() error {
+	startInbound := func(i transport.Inbound) func() error {
 		return func() error {
 			return i.Start(r)
 		}
 	}
 
 	var wait sync.ErrorWaiter
 	for _, i := range r.inbounds {
```

Comment:
should the ordering be the other way around? (e.g. start outbounds first, then inbounds)

the inbound gets an RPC object, could it use that to access an outbound? So it might try to make outbound requests as part of the Start. Since outbounds don't receive the RPC object, they can't have a dependency on any other outbounds/inbounds.